src/util/rank.cpp

```diff
@@ -22,7 +22,10 @@
 #include "rank.h"
 
 #include <assert.h>
+
+#ifndef __aarch64__
```

Comment:
This should "if not x86_64"

Reply:
with this change I think wake will compile on risc-v ;)

Reply:
We should file a task for that haha

Reply:
Send me parts to get my unmatched working and I will do it lol.